src/pages/Staking/index.tsx

```diff
@@ -75,6 +71,37 @@ const convertBlockNumbersToWeeks = (blockNumbers: number) => {
   return blockNumbers * BLOCK_TIME / ONE_WEEK_SECONDS;
 };
 
+// When to increase lock amount and extend lock time
+const checkIncreaseLockAmountAndExtendLockTime = (
+  lockTime: number,
+  lockAmount: GovernanceTokenMonetaryAmount
+) => {
+  return (
```

Comment:
minor (no need to change): bit tidier to use am implicit return in arrow functions when possible